include/chaiscript/language/chaiscript_engine.hpp

```diff
@@ -694,6 +716,57 @@ namespace chaiscript
     T eval_file(const std::string &t_filename, const Exception_Handler &t_handler = Exception_Handler()) {
       return m_engine.boxed_cast<T>(eval_file(t_filename, t_handler));
     }
+
+    // NAMESPACE HANDLING
+    /// \brief Imports a namespace object into the global scope of this ChaiScript instance.
+    void import(const std::string& t_namespace_name)
+    {
+       if (m_namespaces.count(t_namespace_name))

```

Comment:
This code particularly makes my point about `{ }` being always required - mixed `if`/`else if`/`if` without `{ }` can add to confusion during later edits